test_scripts/Smoke/API/014_SetMediaClockTimer_PositiveCase.lua

```diff
@@ -0,0 +1,82 @@
+---------------------------------------------------------------------------------------------------
+-- User story: Smoke
+-- Use case: SetMediaClockTimer
+-- Item: Happy path
+--
+-- Requirement summary:
+-- [SetMediaClockTimer] SUCCESS: getting SUCCESS:UI.SetMediaClockTimer()
+--
+-- Description:
+-- Mobile application sends valid SetMediaClockTimer request and gets UI.SetMediaClockTimer "SUCCESS" response from HMI
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and activated on SDL
+-- c. appID is currently in Background, Full or Limited HMI level
+
+-- Steps:
+-- appID requests SetMediaClockTimer with valid parameters
+
+-- Expected:
+-- SDL validates parameters of the request
+-- SDL checks if UI interface is available on HMI
+-- SDL checks if SetMediaClockTimer is allowed by Policies
+-- SDL checks if all parameters are allowed by Policies
+-- SDL transfers the UI part of request with allowed parameters to HMI
+-- SDL receives UI part of response from HMI with "SUCCESS" result code
+-- SDL responds with (resultCode: SUCCESS, success:true) to mobile application
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonFunctions = require("user_modules/shared_testcases/commonFunctions")
+local commonSmoke = require('test_scripts/Smoke/commonSmoke')
+
+--[[ Local Variables ]]
+local updateMode = {"COUNTUP", "COUNTDOWN", "PAUSE", "RESUME", "CLEAR"}
+
+local requestParams = {
+  startTime = {
+    hours = 0,
+    minutes = 1,
+    seconds = 33
+  },
+  endTime = {
+    hours = 0,
+    minutes = 1 ,
+    seconds = 35
+  }
+}
+
+--[[ Local Functions ]]
+local function SetMediaClockTimer(pParams, pMode, self)
+  local countDown = 0
+  if pMode == "COUNTDOWN" then
+    countDown = -1
```

Comment:
@GetmanetsIrina Why `-1` ?

Reply:
@AByzhynar, in case of "COUNTDOWN" mode, endTime must be less then startTime for validness of request parameters. Will update to setting value according to value in starTime. like 
`if pMode == "COUNTDOWN" then
Parameters.endTime.minutes = Parameters.startTime.minutes -1
end
`
`1` is minimal count step, set as  random value.

Reply:
@AByzhynar , please find update in 76c706f6581df43f9ee00c457e2a7fad7535eeb9